Resources/Prototypes/_SpaceCats/Catalog/Fills/Crates/service.yml

```
@@ -0,0 +1,7 @@
+- type: entity
+  id: CrateServicePrinterInkCartridge
+  parent: CratePlastic
+  components:
+  - type: StorageFill
+    contents:
+      - id: BookPrinterInkCartridge
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)